superset/assets/javascripts/dashboard/v2/containers/DashboardBuilder.jsx

```diff
@@ -7,10 +7,12 @@ import {
   handleComponentDrop,
 } from '../actions/dashboardLayout';
 
-function mapStateToProps({ dashboardLayout: undoableLayout, editMode }) {
+function mapStateToProps({ dashboardLayout: undoableLayout, dashboard }, ownProps) {
   return {
     dashboardLayout: undoableLayout.present,
-    editMode,
+    cells: ownProps.cells,
```

Comment:
why are you overwriting a prop with the same `ownProp`?

Reply:
also see my note above about cells, I think you can just pass these to rows / columns in `containers/DashboardComponent`